pkg/buses/natss/bus_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package natss
+
+import "testing"
+
+func TestStub(t *testing.T) {
```

Comment:
Please add in some tests.

Reply:
@adamharwayne the NATS Streaming server can be embedded inside of the test process to run unit tests again a real NATS Streaming server. This implies vendoring the NATS Streaming server code in Eventing. 
Of course, mocking NATS Streaming could be another solution.
Cause this is the first bus implementation which should add tests, please advise.

Reply:
So far we haven't run the real implementations in unit tests, but it seems like a good idea as long as:

1. It isn't flaky
1. It doesn't take too long (upper bound of 1 minute to start with?).

I didn't realize that the other buses didn't have tests. If you don't vendor the NATS streaming server, then I would focus on testing the event handler functions (e.g. handles errors gracefully, only ACKs the message if the downstream acked, etc.).

Reply:
Thanks! I'll use the embedded NATS Streaming for the unit tests. It's stable and the startup time is aprox. 10s.

Reply:
Done